src/ui/common/src/components/workflows/edges/AqueductBezier.tsx

```diff
@@ -1,24 +1,8 @@
 import React from 'react';
-import { getBezierPath, Position } from 'react-flow-renderer';
-import {theme} from "../../../styles/theme/theme";
+import { EdgeProps, getBezierPath } from 'react-flow-renderer';
+import { theme } from "../../../styles/theme/theme";
 
-type AqueductBezierProps = {
```

Comment:
These are no longer needed and we can just use the built in EdgeProps type that comes with react-flow.